sklearn/metrics/ranking.py

```diff
@@ -259,7 +265,22 @@ def roc_auc_score(y_true, y_score, average="macro", sample_weight=None,
 
     max_fpr : float > 0 and <= 1, optional
         If not ``None``, the standardized partial AUC [3]_ over the range
-        [0, max_fpr] is returned.
+        [0, max_fpr] is returned. For the multiclass case, ``max_fpr``,
+        should be either equal to ``None`` or ``1.0`` as AUC ROC partial
+        computation currently is not supported for multiclass.
+
+    multiclass : string, 'ovr' or 'ovo', optional(default='ovr')
```

Comment:
```suggestion
    multiclass : string, 'ovr' or 'ovo', optional (default='ovr')
```

I do wonder if we should make the user intentionally specify ovo or ovr. Unlike say the average parameter of P/R/F, we don't *need* the user to tell us that the data is to be treated as multiclass (we can use the number of columns in `y_score` for that) but I suspect we should be trying to encourage literacy in the idea that there are several ways to extend roc_auc to multiclass.

Reply:
I am okay with this. We can set the default to `None` and raising an error before going down the multi-class code path.

Reply:
> We can set the default to None and raising an error before going down the multi-class code path.

Do others have opinions on this? @amueller?

Reply:
Since we are moving to strings as keywords more, `multiclass` would most likely be better as `'error'` for raising an exception, or `'warn'` if we decide on just warning.

Also to be consistent with the rest of sklearn, we should use `multi_class`.

Reply:
Let's just do it. We can always change it later to have a default value.